src/main/java/org/openpnp/util/Utils2D.java

```diff
@@ -77,43 +77,33 @@ public static Point scalePoint(Point point, double scaleX, double scaleY) {
 		return new Point(point.getX() * scaleX, point.getY() * scaleY);
 	}
 	
-	public static Location calculateBoardPlacementLocation(Location boardLocation, Side boardSide, Location placementLocation) {
+
+	public static Location calculateBoardPlacementLocation(Location boardLocation, double bottom, Location placementLocation) {
         // We will work in the units of the placementLocation, so convert
         // anything that isn't in those units to it.
         boardLocation = boardLocation.convertToUnits(placementLocation.getUnits());
         
         // If we are placing the bottom of the board we need to invert
         // the placement location.
-        if (boardSide == Side.Bottom) {
-            placementLocation = placementLocation.invert(true, false, false, false);
+        if (bottom != 0.0) {
```

Comment:
this seems a bit strange. either .add with bottom == 0.0 does nothing in the first place, or i don't understand what this is supposed to do.